budget-squirrel/src/pages/Login/Login.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import './Login.scss'
+import BasicForm from '../../components/generic/BasicForm/BasicForm';
+import PasswordInput from '../../controls/PasswordInput/PasswordInput';
+import BasicCheckbox from '../../components/generic/BasicCheckbox/BasicCheckbox';
+
+export default function Login() {
+
+    const title = "Login";
+
+    function LoginInput() {
+        return (
+            <>
+                <input class="input" type="text" placeholder="Username" />
+                <PasswordInput />                
+                <BasicCheckbox title="Remember Me"/>            
+            </>
+        )
+    }
+
+    function LoginButton() {
+        return (
+            <>
+                <button class="button button--wide">Login</button>
+            </>
+        )
+    }
+
+    return (
+        <div class="login">
+            <BasicForm title={title}>
+                <LoginInput key="inputs" />
+                <LoginButton key="buttons" />
+            </BasicForm>
+            <a class="login__forgot-password" href="TBD">
```

Comment:
You don't need to remove this but for future reference, don't add UI for something that isn't part of the task. This task was kind of ambiguous so it's ok but try to stay within the scope of the task. Especially if the UI element doesn't actually do anything yet cause we might forget.